README.md

```diff
@@ -85,6 +86,19 @@ pip install dohq-artifactory==0.5.dev243
 ```
 
 # Usage
+## Enable logging 
+For purpose of debugging or need of additional information you can configure logger to work with current library
+Just configure `logging` module in your python script. Simpliest example:
+~~~python
+import logging
+from artifactory import ArtifactoryPath
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
The example is covered in the standard python documentation.

We should show the case - how to enable debug logging only for the `artifactory | dohq_artifactory` modules and log them into console.

Reply:
same here,
I propose to stick to the same approach as `requests`
they will use global config settings

Reply:
example added to readme